packages/next-ui/Styles/responsiveVal.tsx

```diff
@@ -6,7 +6,7 @@
 export function responsiveVal(
   min: number,
   max: number,
-  maxBreakpoint = 1280,
```

Comment:
Kunnen we bij deze waarde geen `theme.breakpoints.values.xl` gebruiken?

Reply:
responsiveVal doesn't have access to the theme object, so that isn't possible without modifications.

Reply:
It would prevent future bugs that look like this, but might be a bit too much for a simple fix.

Reply:
I can discuss this later how to properly solve it, but for now I'd like to k eep it at 2560